README.md

```diff
@@ -1,2 +1,49 @@
 # in-toto
 in-toto is a series of scripts to protect the supply chain integrity.
+
+# About in-toto
+in-toto guarantees that the end-user (or owner) is able to verify that the entire development life cycle has been conducted as per the specified layout and that each of the functionaries (participants) have perfomed the specified tasks and there haven't been any malicious changes in the files.
```

Comment:
I'd rather not call the end-user "owner", since we are already referring to the actor who writes and signs an in-toto layout as "(project) owner". Also, instead of participants you I'd prefer "(e.g. developers)" or "(carrying out the steps of a software supply chain)".